client/src/components/dashboard/verify/Verify.tsx

```
@@ -89,28 +92,47 @@ const Verify: React.FC = () => {
             </label>
-            {image && <p className="mt-2 text-sm text-green-600">File selected: {image.name}</p>}
-            <button onClick={handleUpload} className="mt-4 px-4 py-2 bg-blue-600 text-white rounded hover:bg-blue-800">
+            {image && (
+              <p className="mt-2 text-sm text-green-600">
+                File selected: {image.name}
+              </p>
+            )}
+            <button
+              onClick={handleUpload}
+              className="mt-4 px-4 py-2 bg-blue-600 text-white rounded hover:bg-blue-800"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)